docs/src/SUMMARY.md

```diff
@@ -11,6 +11,8 @@
 - [Web Apps](cli/web.md)
     - [Default `index.html`](cli/web/default_index_html.md)
 - [Linter](cli/linter.md)
+- [Configuration](cli/configuration.md)
+- [bevy_cli Metadata Reference](cli/metadata-reference.md)
```

Comment:
```suggestion
    - [Configuration Reference](cli/configuration/reference.md)
```

(Nit) I think it would be nicer to nest the reference page under the main "Configuration" page.

I also don't like the "`bevy_lint` Metadata" part of the title because `bevy_lint` is redundant and "Metadata" could be confused for something different than "Configuration". I chose "Configuration Reference", but maybe you have a better name :)

If you do change the title, make sure to update the filename and the `# Title` of the file!